src/model/Tutor.java

```diff
@@ -29,6 +33,10 @@ public void addPet(Pet pet) {
     public List<Pet> getPets() {
         return pets;
     }
+
+    public void unbindPets() {
+        this.pets = null;
+    }

```

Comment:
Isso é uma boa prática mesmo? Se for para permitir deletar um Tutor, a gente vai ter um problema de pets orfãos